Reference/Security/Security-hardening/index.md

```diff
@@ -0,0 +1,59 @@
+# Umbraco Security Hardening
+
+Here you find some tips and trick for hardening the security of your Umbraco installation.
+
+## Rename your Umbraco-folder
+
+By default the login page of Umbraco is available at the path /umbraco/. This page is the entrance to your installation and it’s considered a good practice to rename your path to a more secure path.
+This can be done by following these two steps.
+
+1. Rename you /umbraco/-folder on disk to another path (for example: /my-secret-loginpanel/)
+
+Before:
+
+![Umbraco-folder on disk - before](images/foldersondisk-before.png)
+
+After:
+
+![Umbraco-folder on disk - after](images/foldersondisk-after.png)
+
+2. Change the two keys in your web.config “umbracoReservedPaths” and “umbracoPath” to your new path.
+
+Before:
+
+![Web.config - before](images/webconfig-before.png)
+
+After:
+
+![Web.config - after](images/webconfig-after.png)
+
+From now on, you can only get access to the loginscreen by going to this path and no longer by going to /umbraco/.
+
+## Lock down access to your Umbraco-folders
+
+By default there are some folders in your Umbraco installation that should be only used by authenticated users. It’s considered a good practice to lock down these folders to specific IP-addresses and/or IP-ranges to make sure not everyone can access these.
+The folders we want to lock down are App_Plugins, Config, Umbraco_Client and Umbraco.
+The prerequisite of this to work is that you’re using [IISRewrite](../../routing/iisrewriterules/index.md) 
+
+If you’ve made sure that you’ve installed this on your server we can start locking down our Umbraco-folders. This can be down by following these three steps.
+
+1. We are going to lock down /Umbraco/, but because API-controllers and Surface-controller will use the path /umbraco/api/ and /umbraco/surface/ these will also be locked down. Our first rule in the IISRewrite.config will be used to make sure that these are not locked by IP-address.
+
+	<rule name="Ignore" stopProcessing="true">
+		<match url="^(?:umbraco/api|umbraco/surface)/" />
+		<action type="None" />
+	</rule>
+
+2. Get the IP-addresses of your client and write these down like a regular expression. If the IP-addresses are for example 213.3.10.8 and 88.4.43.108 the regular expression would be ‘213.3.10.8|88.4.43.108”.
+
+3. Lock down the folders App_Plugins, Config, Umbraco_Client and Umbraco (or the renamed version of this folder) by putting this rule into your IISRewrite-rules
+
+	<rule name="Allowed IPs" stopProcessing="true">
+		<match url="^(?:app_plugins|config|umbraco|umbraco_client)(?:/|$)" />
+		<conditions>
+			<add input="{REMOTE_ADDR}" negate="false" pattern="213.3.10.8|88.4.43.108”>
```

Comment:
If an organisation can't identify it's editors by IP range - would it be a reasonable alternative to have an internal dns entry, only on the company's network for the backoffice access?, so have a condition like:

 <!-- specific internal Host address-->
            <add input="{HTTP_HOST}" pattern="backoffice.internaldnsentry.*" negate="false" />

or does that have other side affects or is it more obscurity than security?

Reply:
Never used this, but if you can confirm you've used this before I will add this